Run_2000.py

```diff
@@ -0,0 +1,19 @@
+import time
+
+
+def timer(f: callable, *args) -> float:
+    """
+    A function that receives as a function a function and other parameters.
+    The timer function will measure how long a function f ran when the same
+    parameters are passed to it.
+    :param f: Function
+    :param args: Parameters for the measured function.
+    :return: The running time of f
+    """
+    start = time.time()
+    f(arg for arg in args)
```

Comment:
Don't use abbreviations in variable names  - try to avoid using a single letter variables if possible -
https://softwareengineering.stackexchange.com/questions/72593/how-to-abbreviate-variable-names